analytics/views.py

```diff
@@ -1140,6 +1141,9 @@ def support(request: HttpRequest) -> HttpResponse:
             elif status == "deactivated":
                 do_deactivate_realm(realm, request.user)
                 context["message"] = f"{realm.string_id} deactivated."
+                event_time = timezone_now()
+                RealmAuditLog.objects.create(realm=realm, event_type=RealmAuditLog.REALM_DEACTIVATED,
```

Comment:
Also include the acting `acting_user`. I think it can be the logged-in user who is making the changes in /activity/support.